.styluaignore

```diff
@@ -2,3 +2,4 @@
 /src
 /test
 /runtime/lua/vim/re.lua
+/runtime/lua/vim/_meta/options.lua
```

Comment:
?

Reply:
CI is failing on stylua however adding this line doesn't seem to work.

Reply:
Why is it failing? Is it feasible to fix the warnings?